Class-8-(Intro-to-Spring-and-Controllers).md

```diff
@@ -0,0 +1,34 @@
+# Class 8: Introduction to Spring Boot, MVC, and Controllers
+
+## In the prep work this week, the students learned
+
+1. About the MVC design pattern.
+1. What the Spring framework is.
+1. How to initialize a Spring Boot project.
+1. How to handle routing and write controllers in Spring
+
+## Announcements
```

Comment:
I keep losing track.  When is the assignment 2 due date relative to class 8?  Also, when is the assignment 3 opening date?